packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -146,6 +146,26 @@ function buildLocalCommands(cli, isLocalProject) {
         })
       ),
     })
+    .command({
+      command: "db:sync-links",
+      desc: "Sync database schema with the links defined by your application and Medusa core",
+      builder: (builder) => {
+        builder.option("execute-all", {
+          type: "boolean",
+          describe: "Skip prompts and execute all (including unsafe) actions",
```

Comment:
suggestion: same as for the db:create, I would expect it to be the opposite of skip by default, otherwise in the help it will also show `execute-all: skip...`

Reply:
Did not get. 

The flag says is used when you want to execute all SQL actions and skip prompts. So you will run the command as `npx medusa db:sync-links --execute-all`

Reply:
oh sorry, so execute all and execute safe will always skip the prompt, my bad 😅